src/app/space/create/deployments/services/deployment-api.service.ts

```diff
@@ -17,6 +17,7 @@ import { Observable , of, throwError } from 'rxjs';
 import { catchError , map } from 'rxjs/operators';
```

Comment:
Not introduced or altered in this PR, but this line and the one above it have funny spacing (and an unused import for `of`) if you'd be able to clean that up too it'd be appreciated.